README.md

```diff
@@ -30,10 +31,51 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
       - name: Secret Scanner
-        uses: secret-scanner/action@0.0.2
+        uses: secret-scanner/action@main #or latest release
```

Comment:
Best practice is in fact to do `@full-sha`: 
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Reply:
Noted. I will drop the commit which switches from `0.0.2` to main for now. Then after this is merged update the README to point to the new HEAD commit

Reply:
I think instead of pointing to head you should have `main` list the SHA of the latest release.

Reply:
But we'll probably want a new release with these new features 🙂

Reply:
I just created a new release. https://github.com/secret-scanner/action/pull/16

I think this PR will be my probably be last contribution to this project for the next 2 weeks so I just did this manually for now, but If I start working on this a lot again I will try and write something to bump this automatically whenever a new tag is released